hack/release/pkg/updatecapimate/updatecapimate_test.go

```diff
@@ -0,0 +1,49 @@
+package updatecapimate
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/mesosphere/kommander-applications/hack/release/pkg/constants"
+	cp "github.com/otiai10/copy"
+	"github.com/stretchr/testify/assert"
+)
+
+const rootDir = "../../../../"
+
```

Comment:
To increase the test coverate
```suggestion

func TestUpdateCAPIMateVersionsFailsWhenItCannotFindCM(t *testing.T) {
	tmpDir, err := os.MkdirTemp("", "prerelease")
	assert.Nil(t, err)
	defer os.RemoveAll(tmpDir)

	updateToVersion := "v1.0.0"
	err = UpdateCAPIMateVersion(tmpDir, updateToVersion)
	assert.ErrorContains(t, err, "verify the kommander-applications repo path")
}

```

Reply:
Thanks!